### ISSUE TITLE
Sports teams in test split

### ISSUE THREAD

**Issue by alanakbik** [CODAIT/Identifying-Incorrect-Labels-In-CoNLL-2003#38]
Again thanks for sharing this work. The annotations look much improved over the original CoNLL. Especially sports teams that originally were not well labeled are now much better. 

However, is seems that some sports teams in the test split are still labeled as LOC, but I think they should be ORG. For example, in the first few sentences we see a couple of instances like this: 

```
SOCCER NN I-NP O
- : O O
JAPAN NNP I-NP I-ORG
GET VB I-VP O
LUCKY NNP I-NP O
WIN NNP I-NP O
, , O O
CHINA NNP I-NP I-LOC
IN IN I-PP O
SURPRISE DT I-NP O
DEFEAT NN I-NP O
. . O O

But CC O O
China NNP I-NP I-ORG
saw VBD I-VP O
their PRP$ I-NP O
luck NN I-NP O
desert VB I-VP O
them PRP I-NP O
in IN I-PP O
the DT I-NP O
second NN I-NP O
match NN I-NP O
of IN I-PP O
the DT I-NP O
group NN I-NP O
, , O O
crashing VBG I-VP O
to TO I-PP O
a DT I-NP O
surprise NN I-NP O
2-0 CD I-NP O
defeat NN I-NP O
to TO I-PP O
newcomers NNS I-NP O
Uzbekistan NNP I-NP I-LOC
. . O O

Japan NNP I-NP I-LOC
coach NN I-NP O
Shu NNP I-NP I-PER
Kamo NNP I-NP I-PER
said VBD I-VP O
...
```

In these examples "CHINA", "Uzbekistan" and "Japan" should be ORG (the other teams "JAPAN" and "China" in this example are labeled as ORG).

**Comment by frreiss** [comment]
Thanks for catching those problems in the first document of the test fold! I must confess that I went over that document by hand at least twice and managed to miss those additional corrections :-\

The instance of `CHINA` in the title being tagged `LOC` was due to there being conflicting manual corrections. The original tag for that token was `PER`. One file corrected it to `LOC`, and another corrected it to `ORG`. I've changed the audited files so that both change that tag to `ORG`.

The remaining corrections just weren't caught by our ensembles of models, and I guess our eyes skated over them while we were looking at the problems that *were* flagged by the models. Go figure.

I've fixed all of these in #39.

Here are the corrections for the first document in the test fold BEFORE those additional changes:
![image](https://user-images.githubusercontent.com/12436991/106503725-e07ce000-647a-11eb-835f-4100952f9008.png)

And here is the same file AFTER, with new corrections in red and the single modified tag in blue:
![image](https://user-images.githubusercontent.com/12436991/106503737-e4106700-647a-11eb-8049-d42f262e84b9.png)

Please let us know if you see anything additional problems we missed!

**Comment by frreiss** [comment]
Fixed in #39

**Comment by alanakbik** [comment]
@frreiss awesome, thanks for correcting this!
